File: HPSC 2/mains.py
import time
from timecons import timeconsumingfunc
n=300
start_time =time.time()
for i in range(n):
    timeconsumingfunc()
end_time = time.time()
exe= end_time - start_time
print(f'The excecution time is : {exe:.4f}')

File: HPSC 2/mains.R
library(foreach)
timeconsumingfun <- function() {
  
  Sys.sleep(5)  
}

start_time <- proc.time()[3]
n<-150
foreach(i = 1:n, .combine = c) %do% {
  timeconsumingfun()
}

end_time <- proc.time()[3]
execution_time <- end_time - start_time
print(paste("Execution time:", execution_time))

